tasks/agent.yml

```diff
@@ -66,6 +66,19 @@
   with_items:
     "{{ bamboo_agent.capabilities }}"
   when: (item.source is defined and item.source == "repository")
+  when: >
```

Comment:
Can you use lists instead, which improves readability?

```
  when:
    - item.source is defined
    - item.source == "repository"
    - (ansible_distribution == 'CentOS' or ansible_distribution == 'Red Hat Enterprise Linux')
```

Reply:
That's fine.